spec/rubocop/cop/style/one_line_conditional_spec.rb

```diff
@@ -5,9 +5,14 @@
 describe RuboCop::Cop::Style::OneLineConditional do
   subject(:cop) { described_class.new }
 
-  it 'registers an offense for one line if/then/end' do
+  it 'registers an offense for one line if/then/else/end' do
     inspect_source(cop, 'if cond then run else dont end')
     expect(cop.messages).to eq(['Favor the ternary operator (?:)' \
                                 ' over if/then/else/end constructs.'])
   end
+
+  it 'does not register an offense for if/then/end' do
```

Comment:
Yeah, sure. I was just thinking this is not exactly related to the bug you wanted to solve.